### PR TITLE
Status development tools: HTTP debugging, updated react-native-http-bridge

### PR DESCRIPTION
**Pull request by alwx**
Status: ready

(re-)Introduces the debug bridge for Status. Also uses the new and updated version of `react-native-http-bridge`, the work for it was done here: https://github.com/alwx/react-native-http-bridge

Description of HTTP server and requests server can handle: https://gist.github.com/alwx/51d2758a9d82b392c6f46eff2e34cde3

<blockquote><div> GitHub</div><div><strong><a href="https://github.com/alwx/react-native-http-bridge">alwx/react-native-http-bridge</a></strong></div><div>HTTP server for React Native. Contribute to alwx/react-native-http-bridge development by creating an account on GitHub.</div></blockquote>

### CODE REVIEW

**Comment by yenda** [comment]
@alwx The branch needs to be rebased, also it seems the http-server is always started shouldn't it be a developer option only ?

(edit: sorry I just saw it is a wip)

File: src/status_im/debug/events.cljs [hunk 1]
```diff
@@ -0,0 +1,98 @@
+(ns status-im.debug.events
```
Comment (flexsurfer) [inline src/status_im/debug/events.cljs]: I'm not sure why debug? maybe something like `dev-server` ?

File: src/status_im/debug/events.cljs [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+
+;; FX
+
+(re-frame/reg-fx
+ :http-server/start
+ (fn []
+   (http-server/start!)))
+
+(re-frame/reg-fx
+ :http-server/start-if-needed
```
Comment (flexsurfer) [inline src/status_im/debug/events.cljs]: `(data-store.accounts/get-by-address` looks weird in `fx` usually, we pass all data from event, you can get account from cofx and pass it here instead address

File: src/status_im/debug/events.cljs [hunk 3]
```diff
@@ -0,0 +1,98 @@
...
+  (models.network/connect
+   cofx
+   {:network    (:id data)
+    :on-success (fn [network _]
+                  {:http-server/respond [200 {:message    "Network has been connected."
+                                              :network-id network}]})
+    :on-failure (fn [_ _]
+                  {:http-server/respond [400 {:message "The network id you provided doesn't exist."}]})}))
+
+(defmethod process-request! [:DELETE "network" nil]
```
Comment (flexsurfer) [inline src/status_im/debug/events.cljs]: why not move multimethods in model?

File: src/status_im/init/core.cljs [hunk 4]
```diff
@@ -163,9 +163,11 @@
                             :web3/fetch-node-version     [web3
                                                           #(re-frame/dispatch
                                                             [:web3/fetch-node-version-callback %])]
+                            :http-server/start-if-needed address
                             :notifications/get-fcm-token nil}
                            (initialize-account-db address)
                            (models.protocol/initialize-protocol address)
+
```
Comment (flexsurfer) [inline src/status_im/init/core.cljs]: extra line

**Review by flexsurfer — CHANGES_REQUESTED** [review]

**Comment by alwx** [comment]
@flexsurfer the PR was updated

**Review by flexsurfer — APPROVED** [review]

**Review by jeluard — APPROVED** [review]

**Pull request merged**